cli/MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+include src/semgrep/rule_schema.yaml
+include src/semgrep/lang/lang.json
+include LICENSE
+include src/semgrep/templates/.semgrepignore
```

Comment:
this didn't get recognized as a rename because I also had to change 3 out of 4 lines